### PR TITLE
Adds new Asteroid City theme

### PR DESCRIPTION
**Pull request by MelanieCrissey**
# Pull Request Template

## Discord username (optional)

melaniecrissey

## Name of theme

Asteroid City

## How Has This Been Tested?

Have you run the python script? `python3 ./scripts/gen_theme_previews.py <folder-name-eg-standard>`
[Yes]

## Notes

We cannot accept pull request that include custom background images because:

- of licensing restrictions
- we are trying to keep the binary size of the repo as small as possible (just the yaml files)

If your theme has an intended custom background image, include a comment in the yaml with a link to where people should download it.

Consider making an exception for Special Edition? ;D

### CODE REVIEW

File: special_edition/asteroid_city.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+accent: '#D83E1C'
+background: '#c4f4f1'
+details: darker
+foreground: '#22211a'
+
+############################################################### SEE BELOW
+background_image:
+  # the path is relative to ~/.warp/themes/
+  # the full path to the picture is: ~/.warp/themes/warp.jpg
+  path: ~/.warp/themes/asteroid_city/asteroid_city_wallpaper.jpg
```
Comment (elviskahoro) [inline special_edition/asteroid_city.yaml:10]: You wanna use a relative path not the absolute:

`special_edition/asteroid_city_wallpaper.jpg'

**Review by elviskahoro — CHANGES_REQUESTED** [review]

File: special_edition/asteroid_city.yaml (hunk 1 above)
Comment (MelanieCrissey) [reply special_edition/asteroid_city.yaml]: Done! Made the change. It doesn't work for me locally, but I'm assuming it'll work as expected when this image gets included in the system folder?

Comment (elviskahoro) [reply special_edition/asteroid_city.yaml]: I'll try to debug this and see what's up

**Review by hongyi-chen — CHANGES_REQUESTED** [review]
Awesome stuff! You'll also need to add the .svg preview to `special_edition/README.md`, similar to how it is done here: https://github.com/warpdotdev/themes/pull/65/files

File: special_edition/asteroid_city.yaml (hunk 1 above)
Comment (hongyi-chen) [reply special_edition/asteroid_city.yaml]: I just tried this locally and it seems to work alright for me! Not sure why this doesn't work for Melanie.

image

File: special_edition/asteroid_city.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+background: '#c4f4f1'
+details: darker
+foreground: '#22211a'
+
+############################################################### SEE BELOW
+background_image:
+  # the path is relative to ~/.warp/themes/
+  # the full path to the picture is: ~/.warp/themes/warp.jpg
+  path: special_edition/asteroid_city_wallpaper.jpg
+  # the opacity value is required and can range from 0-100
```
Comment (hongyi-chen) [inline special_edition/asteroid_city.yaml:11]: nit: might be best to link the source of the image here, just so we don't run into any copyright issues

**Comment by elviskahoro** [comment]
> Awesome stuff! You'll also need to add the .svg preview to `special_edition/README.md`, similar to how it is done here: https://github.com/warpdotdev/themes/pull/65/files

I can run the script and add them.

**Pull request merged**